modules/virt-customizing-storage-profile-default-cloning-strategy.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * virt/storage/virt-configuring-storage-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-customizing-storage-profile-default-cloning-strategy_{context}"]
+= Setting a default cloning strategy by using a storage profile
+
+You can use storage profiles to set a default cloning method for a storage class, creating a _cloning strategy_. Setting cloning strategies can be helpful, for example, if your storage vendor only supports certain cloning methods. It also allows you to select a method that limits resource usage or maximizes performance.
+
+Cloning strategies can be specified by setting the `cloneStrategy` attribute in a storage profile to one of these values:
+
+* `snapshot` is used by default when snapshots are configured. The CDI will use the snapshot method if it recognizes the storage provider and the provider supports Container Storage Interface (CSI) snapshots. This cloning strategy uses a temporary volume snapshot to clone the volume.
+* `copy` uses a source pod and a target pod to copy data from the source volume to the target volume. Host-assisted cloning is the least efficient method of cloning.
+* `csi-clone` uses the CSI clone API to efficiently clone an existing volume without using an interim volume snapshot. Unlike `snapshot` or `copy`, which are used by default if no storage profile is defined, CSI volume cloning is only used when you specify it in the `StorageProfile` object for the provisioner's storage class.
+
+[NOTE]
+====
+You can also set clone strategies using the CLI without modifying the default `claimPropertySets` in your YAML `spec` section.
```

Comment:
```suggestion
You can set clone strategies using the CLI without modifying the default `claimPropertySets` in your YAML `spec` section.
```

Reply:
I didn't update this content, but I've made the change.